packages/peer-store/test/merge.spec.ts

```diff
@@ -244,4 +241,53 @@ describe('merge', () => {
     expect(updated).to.have.property('tags').that.deep.equals(original.tags)
     expect(updated).to.have.property('protocols').that.deep.equals(original.protocols)
   })
+
+  it('merges addresses', async () => {
```

Comment:
```suggestion
  it('merges lastFailure & lastSuccess on peer addresses', async () => {
```

Reply:
This test also ensures `multiaddr` and `isCertified` values are merged and not lost so they are missing from the test name if you want it to be exhaustive, or it can remain as it is.